src/applications/vaos/appointment-list/components/NoAppointments.jsx

```diff
@@ -14,13 +15,25 @@ export default function NoAppointments({
       </h3>
       {showScheduleButton && (
         <>
-          <p>
-            You can schedule an appointment online now, or call your{' '}
-            <NewTabAnchor href="/find-locations">
-              VA medical center
-            </NewTabAnchor>{' '}
-            to schedule an appointment.
-          </p>
+          {additionalRequestDescription ? (
+            <p>
+              If you request an appointment it will show here until staff review
+              and schedule it. You can schedule an appointment online now, or
+              call your{' '}
+              <NewTabAnchor href="/find-locations">
+                VA health facility
+              </NewTabAnchor>{' '}
+              to schedule an appointment.
+            </p>
+          ) : (
+            <p>
+              You can schedule an appointment online now, or call your{' '}
+              <NewTabAnchor href="/find-locations">
+                VA medical center
```

Comment:
@young1205, I spoke to Peter yesterday and he confirmed all instances of "VA medical center"  should be changed to "VA health facility" for the no appointment alert message, 
cc: @outerpress

Reply:
fixed so both say `VA health facility`